src/components/atoms/RedirectButton.tsx

```diff
@@ -7,11 +7,14 @@ import { AvailablePortalRoutes } from '@/types/copilotPortal'
 interface RedirectButtonProps {
   route: AvailablePortalRoutes
   children: string | ReactNode
+  execute: boolean
 }
 
-const RedirectButton = ({ route, children }: RedirectButtonProps) => {
+const RedirectButton = ({ route, children, execute }: RedirectButtonProps) => {
   const handleClick = () => {
-    window.parent.postMessage({ type: 'history.push', route }, '*')
+    if (execute) {
+      window.parent.postMessage({ type: 'history.push', route }, '*')
+    }
```

Comment:
Not really a big issue but maybe we could name the prop something like `disabled` since that prop is more commonly used in context of buttons